specification/web/resource-manager/Microsoft.Web/2016-08-01/WebApps.json

```diff
@@ -1913,6 +1913,252 @@
         }
       }
     },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Web/sites/{name}/continuouswebjobs": {
+      "get": {
+        "tags": [
+          "WebApps"
+        ],
+        "summary": "List continuous web jobs for an app, a specific deployment slot, and/or a specific scaled-out instance.",
+        "description": "List continuous web jobs for an app, a specific deployment slot, and/or a specific scaled-out instance.",
+        "operationId": "WebApps_ListContinuousWebJobs",
+        "produces": [
+          "application/json",
+          "text/json"
```

Comment:
Needs to be fixed all up. I will have that fixed as a separate PR across the board. 

---
In reply to: [133797492](https://github.com/Azure/azure-rest-api-specs/pull/1551#discussion_r133797492) [](ancestors = 133797492)